molo/core/views.py

```diff
@@ -10,3 +12,44 @@ def locale_set(request, locale):
 
 def health(request):
     return HttpResponse(status=200)
+
+
+def versions(request):
+    comparison_url = "https://github.com/praekelt/%s/compare/%s...%s"
+    packages = [('molo.core', 'Molo'), ('molo.profiles', 'Profiles'),
+                ('molo.commenting', 'Commenting'), ('molo.polls', 'Polls'),
+                ('molo.yourwords', 'YourWords'), ('molo.servicedirectory',
+                'Srvice Directory')]
```

Comment:
@miltontony is there a better way of doing this? simon suggested to I get the packages from the requirement.txt but then It won't show the packages that are not installed and It's gonna show all the packages in there which maybe for this I can loop through packages that start with molo only.